docs/unified-test-documentation/dasharo-security/20T-network-boot-availability.md

```diff
@@ -0,0 +1,77 @@
+# Dasharo Security: Network Boot availability
```

Comment:
@Pokisiekk, we have some inconsistency here. In our products documentation, we had the product: `Network stack enable/disable`. I think, that in documentation we should use the same nomenclature

Reply:
Renamed